src/coreclr/jit/llvm.h

```diff
@@ -38,8 +38,13 @@ struct OperandArgNum
 
 struct LlvmArgInfo
 {
-    int m_argIx; // -1 indicates not in the LLVM arg list, but on the shadow stack
+    int          m_argIx; // -1 indicates not in the LLVM arg list, but on the shadow stack
     unsigned int m_shadowStackOffset;
+
+    bool         IsLlvmArg()
```

Comment:
```suggestion
    bool IsLlvmArg()
```
(Fun fact: this is how alignment used to be in the Jit codebase before clang formatting)